Side-Projects.md

```diff
@@ -21,24 +21,30 @@ to your host computer's local disk
 
 ```bash
 cd <directory with docker-compose.yml>
-docker-compose up -d 
+docker compose up -d 
 ```
 
+Now as long as you don't `docker compose down`, the docker daemon (`sudo systemctl enable docker`) will automatically start the various containers every time the Pi boots.
+
+**Logs:**
+
+Run `docker attach unifi_logs` to tail the various unifi logs files.
```

Comment:
why would you want to "attach" to this container?

Reply:
That's the only way I could figure out how to access the log files (to diagnose issues). Is there another way besides `attach`?